src/core/inlines.js

```diff
@@ -70,66 +136,22 @@ export function run(conf) {
       if (subtxt.length) matched = subtxt.shift();
       df.appendChild(document.createTextNode(t));
       if (matched) {
-        // RFC 2119
         if (
           /MUST(?:\s+NOT)?|SHOULD(?:\s+NOT)?|SHALL(?:\s+NOT)?|MAY|(?:NOT\s+)?REQUIRED|(?:NOT\s+)?RECOMMENDED|OPTIONAL/.test(
             matched
           )
         ) {
-          matched = matched.split(/\s+/).join(" ");
-          df.appendChild(
-            hyperHTML`<em class="rfc2119" title="${matched}">${matched}</em>`
-          );
-          // remember which ones were used
-          rfc2119Usage[matched] = true;
+          const node = inlineRFC2119Matches(matched);
+          df.appendChild(node);
```

Comment:
I wonder if we should instead structure this section like:

```JS
if ( condition ) {
   const node = inlineRFC2119Matches(matched);
   df.appendChild(node);
   continue;
}
if (matched.startsWith("{{{")) {
   const node = inlineXrefMatches(matched);
   df.appendChild(node);
   continue;
}
// And so on...
```

Reply:
I'd prefer `else if` because it explicitly means the branches are mutually exclusive.

Reply:
ok, yeah... I was kinda abusing "continue;" as meaning that :)